nixos/modules/services/networking/charybdis.nix

```diff
@@ -21,30 +21,30 @@ in
       enable = mkEnableOption "Charybdis IRC daemon";
 
       config = mkOption {
-        type = types.string;
+        type = types.str;
```

Comment:
```suggestion
        type = types.lines;
```

Reply:
but I'm not sure the service supports multiple elements in config.

Reply:
Yeah it doesn't, I checked, see https://charybdis.readthedocs.io/en/release-3.5/config.html